community-membership.md

```diff
@@ -1,73 +1,108 @@
+Committers come from Reviewers. Committer is scoped to the specific SIG where the promotion applies. Committers are recoganized by Maintainers or SIG Tech Leads for their outstanding contributions to the specific SIG.
+
+Committer status is scoped to the part of the codebase under the SIG.
 
 ### How to become a Committer:
 
-- Demonstrated good technical judgment
-- Assumed the Reviewer role in at least two or more subprojects
-- Reviewed or merged a certain amount of TiKV code
-- Contributed significantly to TiKV performance improvements, or fix major bugs
-- Voted by two or more incumbent Maintainers
+- Demonstrated a good sense of responsibility in PR reviews within the SIG. The review contributions must meet the amount or the difficulty criteria as defined in the charter of the specific SIG.
+- Demonstrated deep understanding of the TiKV components specific to the SIG by contributing significantly to TiKV performance improvements, or fix major bugs, which can be achieved by satisfying one or more requirements below:
+    - Finished 2+ tasks of Medium difficulty or higher
+    - Fixed 1+ task of Hard difficulty
+- Nominated by the Tech Leads of the SIG and gained supermajority (2/3) votes from Maintainers
+
+**Notes:**
+>
+> - A review is only counted when it provides valid comments or edits that advances the PR to approval and merging.
+> - PR characteristics may vary among SIGs, so the requirements for PR reviews are subject to the specific SIG.
 
 ### Privileges and responsibilities:
 
-- Control overall code quality of the project
+- Control overall code quality of projects under the SIG
 - Guide Contributors and Reviewers to contribute to TiKV continuously
-- Participate in Committer meetings on a regular basis and have knowledge priority regarding project development
+- Participate in Committer meetings regularly and have knowledge priority regarding project development
+- Participate in design discussions
+- Get the profile listed on the membership page of the corresponding SIG
 
 ## Reviewer
 
-Reviewers are responsible for reviewing the code submitted by the community to ensure the quality and correctness; generated out of Contributors.
+Reviewers are responsible for reviewing the code submitted by the community to ensure the quality and correctness; generated out of Active Contributors in a specific SIG.
+
+Reviewer status is scoped to the part of the codebase under the SIG.
 
 ### How to become a Reviewer:
 
 - Familiar with the codebase
-- Contributed a certain amount of code to TiKV subprojects
-- Performed excellently in a Special Interest Group (SIG)
-- Sponsored by at least two incumbent Committers
+- Demonstrated a recoganized level of community engagement in the SIG development, via contributions that have met one or more of the minimal criteria as defined below：
+  - Merged 20+ PRs (may vary among SIGs)
```

Comment:
20+ PRs is too many?

Reply:
I am not sure. How many do you propose? 10? And it may be different for each SIG